src/main/cljs/dmohs/react/core.cljs

```diff
@@ -154,6 +154,12 @@
 (defn after-update [instance f & args]
   (.setState instance #js{} (fn [] (apply f args))))
 
+(defn force-update
+  ([instance]
+   (.forceUpdate instance))
+  ([instance f]
+   (.forceUpdate instance f)))
```

Comment:
If you use the same pattern as `after-update` for this, users can:
```clojure
(force-update this :do-stuff "my-argument-1" "my-argument-2")
```
which is a common pattern for `after-update`.

Reply:
I've mainly seen it as 
```cljs
(after-update #(this :do-stuff "my-argument-1" "my-argument-2")
```

Reply:
The first iteration didn't follow this pattern, which is probably why you see that usage in FC. In my other projects, the second style is the norm.